content/pt-br/docs/reference/scheduling-policies/policies.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Políticas do escalonador
+content_type: concept
+sitemap:
+  priority: 0.2 # Scheduling priorities are deprecated
```

Comment:
There's a missing `weight: 30` attribute at the end of this front matter content.